### ISSUE TITLE
[BUG] Happens when scrolling, no other addons enabled

### ISSUE THREAD

**Issue by JoeyJFranklin** [Eliote/SimpleAddonManager#28]
```lua
Message: ...terface/AddOns/SimpleAddonManager/AddonListFrame.lua:321: bad argument #1 to 'GetAddOnMetadata' (Invalid AddOn name ElvUI. - Usage: local value = C_AddOns.GetAddOnMetadata(name, variable))
Time: Mon May 22 16:42:40 2023
Count: 1
Stack: ...terface/AddOns/SimpleAddonManager/AddonListFrame.lua:321: bad argument #1 to 'GetAddOnMetadata' (Invalid AddOn name ElvUI. - Usage: local value = C_AddOns.GetAddOnMetadata(name, variable))
[string "=[C]"]: in function `GetAddOnMetadata'
[string "@Interface/AddOns/SimpleAddonManager/AddonListFrame.lua"]:321: in function `update'
[string "@Interface/AddOns/SimpleAddonManager/Filters.lua"]:305: in function `UpdateListFilters'
[string "@Interface/AddOns/SimpleAddonManager/Core.lua"]:224: in function `Update'
[string "@Interface/AddOns/SimpleAddonManager/Core.lua"]:231: in function <Interface/AddOns/SimpleAddonManager/Core.lua:229>
[string "=[C]"]: ?
[string "=[C]"]: in function `Show'
[string "@Interface/FrameXML/UIParent.lua"]:3246: in function `ShowUIPanel'
[string "@Interface/AddOns/SimpleAddonManager/Core.lua"]:432: in function <Interface/AddOns/SimpleAddonManager/Core.lua:428>

Locals: (*temporary) = "ElvUI"
(*temporary) = "Version"
(*temporary) = "Invalid AddOn name ElvUI. - Usage: local value = C_AddOns.GetAddOnMetadata(name, variable)"
```

**Comment by JoeyJFranklin** [comment]
I think it happens if you have the options "View AddOn list as dependency tree" and "Show Versions in AddOns list" selected. Have one or the other selected doesn't produce the bug but if I have both selected and start scrolling up and down I get the bug.

**Comment by Eliote** [comment]
Blizzard moved `GetAddOnMetadata` API to a new namespace (C_AddOns). I'll try to release a fix tomorrow.

Thanks!

**Comment by Eliote** [comment]
Fixed by 039f31b261022cb7fae0ab9c088cec0205195483
